polar-core/src/vm.rs

```diff
@@ -535,7 +535,7 @@ impl PolarVirtualMachine {
             Goal::TraceRule { trace } => {
                 if let Node::Rule(rule) = &trace.node {
                     self.log(
-                        LogLevel::Trace,
+                        LogLevel::Info,
```

Comment:
Is this going to be redundant with the `QUERY RULE` log from L1485?

Reply:
I don't believe so. The L1485 event is emitted at the top of the query for the outer call to the rule w/ the arguments whereas this L538 entry shows us entering into the body of a specific rule definition w/ accompanying source.